mobile/components/MutationWrapper/index.js

```diff
@@ -0,0 +1,37 @@
+// @flow
+import * as React from 'react';
+
+type Props = {
+  mutation: Function,
+  variables: any,
+  render: Function,
+};
+
+type State = {
+  isLoading: boolean,
+};
+
+class MutationWrapper extends React.Component<Props, State> {
```

Comment:
Some docs at the top of this file as to why this is useful would be ace

Reply:
Surely